Don't build project

The project don't build with cmake.

message error:

```
ubuntu:@user~/Dpubsub/build$ cmake ..
CMake Error at examples/tcp/CMakeLists.txt:3 (find_package):
  By not providing "Finddpubsub.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "dpubsub", but
  CMake did not find one.

  Could not find a package configuration file provided by "dpubsub" with any
  of the following names:

    dpubsubConfig.cmake
    dpubsub-config.cmake

  Add the installation prefix of "dpubsub" to CMAKE_PREFIX_PATH or set
  "dpubsub_DIR" to a directory containing one of the above files.  If
  "dpubsub" provides a separate development package or SDK, be sure it has
  been installed.


-- Configuring incomplete, errors occurred!
See also "/home/alunoifpe/Dpubsub/build/CMakeFiles/CMakeOutput.log".
```

```
ubuntu:@user~/Dpubsub/build$ cmake ..
CMake Error at tests/CMakeLists.txt:3 (find_package):
  By not providing "Finddpubsub.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "dpubsub", but
  CMake did not find one.

  Could not find a package configuration file provided by "dpubsub" with any
  of the following names:

    dpubsubConfig.cmake
    dpubsub-config.cmake

  Add the installation prefix of "dpubsub" to CMAKE_PREFIX_PATH or set
  "dpubsub_DIR" to a directory containing one of the above files.  If
  "dpubsub" provides a separate development package or SDK, be sure it has
  been installed.


-- Configuring incomplete, errors occurred!
See also "/home/alunoifpe/Dpubsub/build/CMakeFiles/CMakeOutput.log".

```

--- a/examples/tcp/CMakeLists.txt
+++ b/examples/tcp/CMakeLists.txt
@@ -1,7 +1,6 @@
 cmake_minimum_required(VERSION 3.10)
-project(myapp)
-find_package(dpubsub REQUIRED)
+project(mqtt_tcp_api_example)
 
-add_executable(my_app mqtt_tcp.c)
-target_link_libraries(my_app PRIVATE dpubsub)
+add_executable(mqtt_tcp_api_example mqtt_tcp.c)
+target_link_libraries(mqtt_tcp_api_example PRIVATE dpubsub)
 

--- a/examples/tcp/mqtt_tcp.c
+++ b/examples/tcp/mqtt_tcp.c
@@ -10,7 +10,7 @@ int main() {
     
     char *client_id = "mqtt_TCP_client";
     uint16_t keep_alive_interval = 60; 
-    char *ip_broker = "35.172.255.228";
+    char *ip_broker = "35.172.255.228"; //endereco do emqx
     uint16_t port_broker = 1883;
     uint8_t default_qos = 0;
     char *topic = "dpubsub";

--- a/tests/CMakeLists.txt
+++ b/tests/CMakeLists.txt
@@ -1,6 +1,5 @@
 cmake_minimum_required(VERSION 3.10)
 project(tests)
-find_package(dpubsub REQUIRED)
 
 add_executable(client_test client_test.c)
 add_executable(api_test api_test.c)

